languages/ruby/exercises/concept/numbers/.meta/design.md

```diff
@@ -27,6 +27,7 @@ The Concepts this exercise unlocks are:
 This exercise's prerequisite Concepts are:
 
 - `basics`: know how to define methods.
+- `booleans`: know about `true`/`false` and truthy/falsy values.
```

Comment:
@dvik1950 Could you also update the `languages/ruby/config.json` file to have this new prerequisite? It is the `languages/ruby/config.json` that the v3 website will use to get the concept and prerequisite information for exercises. The `design.md` document is just there as documentation.

Reply:
Sorry, no idea how I missed that one